vdr/api/v1/api.go

```diff
@@ -79,9 +80,47 @@ func (a *Wrapper) Routes(router core.EchoRouter) {
 
 // CreateDID creates a new DID Document and returns it.
 func (a Wrapper) CreateDID(ctx echo.Context) error {
-	doc, err := a.VDR.Create()
+	req := DIDCreateRequest{}
+	if err := ctx.Bind(&req); err != nil {
+		return ctx.String(http.StatusBadRequest, fmt.Sprintf("create request could not be parsed: %s", err.Error()))
+	}
+
+	options := doc2.DefaultCreationOptions()
```

Comment:
`doc2` is a bit ugly and non-descriptive

Reply:
changed to `vdrDoc`